conformance/utils/suite/suite.go

```diff
@@ -129,12 +150,14 @@ func New(s Options) *ConformanceTestSuite {
 		roundTripper = &roundtripper.DefaultRoundTripper{Debug: s.Debug, TimeoutConfig: s.TimeoutConfig}
 	}
 
-	if s.SupportedFeatures == nil {
+	if s.EnableAllSupportedFeatures == true {
+		s.SupportedFeatures = AllFeatures
+	} else if s.SupportedFeatures == nil {
 		s.SupportedFeatures = StandardCoreFeatures
 	} else {
-		for feature, val := range StandardCoreFeatures {
-			if _, ok := s.SupportedFeatures[feature]; !ok {
-				s.SupportedFeatures[feature] = val
+		for feature := range StandardCoreFeatures {
+			if !s.SupportedFeatures.Has(feature) {
+				s.SupportedFeatures.Insert(feature)
```

Comment:
nit: now that we're using a set, you could just do an `Insert(feature)` here for every element in the `StandardCoreFeatures`, as duplicates wont matter. Not necessary though.